content/2_ModuleTwo_Setup/_index.ee.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "AWS Hosted Workshop Setup" # MODIFY THIS TITLE IF APPLICABLE
+chapter: true # MAKE SURE YOU SET THIS TO FALSE IF USING SELF-GUIDED SETUP
+weight: 2
+---
+
+# AWS Hosted Workshop Setup <!-- MODIFY THIS HEADING -->
+
+## Module Two Heading <!-- MODIFY THIS SUBHEADING -->
+
+This paragraph block should be an introduction to the module about the method of setting up the workshop using EventEngine in AWS. Some typical examples of the steps in this process will explain logging out of any current AWS accounts, how to utilize the hashed URL, utilizing the Team Dashboard to log in to the AWS console, and ensure that the correct region is being selected.
```

Comment:
Content guidance can be viewed here: https://circleci.awsworkshop.io/ee/020_event_engine_setup.html